### PR TITLE
[PT FE]: support aten:Bool, add tests for aten::add_

### PR DESCRIPTION
**Pull request by eaidova**
### Details:
 - *support aten::Bool operator*
 - *Added test cases for aten::add_*

### Tickets:
 - *10110*
 - *101100*

### CODE REVIEW

File: src/frontends/pytorch/src/op/bool.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+namespace op {
+
+OutputVector translate_bool(NodeContext& context) {
+    return {context.mark_node(std::make_shared<ov::op::v0::Convert>(context.get_input(0), element::boolean))};
+};
+
+}  // namespace op
+}  // namespace pytorch
+}  // namespace frontend
+}  // namespace ov
```
Comment (eaidova) [inline src/frontends/pytorch/src/op/bool.cpp:20]:
```suggestion
}  // namespace ov

```

File: tests/layer_tests/pytorch_tests/test_bool.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+        ref_net = None
+
+        return prim_bool(input_type), ref_net, "aten::Bool"
+
+    @pytest.mark.parametrize("input_type", ["tensor", "scalar"])
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    def test_ceil(self, ie_device, precision, ir_version, input_type):
+        self._test(*self.create_model(input_type), ie_device, precision, ir_version)
```
Comment (eaidova) [inline tests/layer_tests/pytorch_tests/test_bool.py:36]:
```suggestion
        self._test(*self.create_model(input_type), ie_device, precision, ir_version)

```

**Review by mvafin — APPROVED** [review]

File: src/frontends/pytorch/src/op/bool.cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/convert.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_bool(NodeContext& context) {
+    return {context.mark_node(std::make_shared<ov::op::v0::Convert>(context.get_input(0), element::boolean))};
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/bool.cpp:14]: Please check number of inputs

**Review by mvafin — CHANGES_REQUESTED** [review]

File: src/frontends/pytorch/src/op/bool.cpp (hunk 3 above)
Comment (eaidova) [reply src/frontends/pytorch/src/op/bool.cpp:17]: added

**Review by mvafin — APPROVED** [review]

**Pull request merged**